RELEASE.md

```diff
@@ -9,10 +9,9 @@ The Kubespray Project is released on an as-needed basis. The process is as follo
 5. An approver creates [new release in GitHub](https://github.com/kubernetes-sigs/kubespray/releases/new) using a version and tag name like `vX.Y.Z` and attaching the release notes
 6. An approver creates a release branch in the form `release-X.Y`
 7. The corresponding version of [quay.io/kubespray/kubespray:vX.Y.Z](https://quay.io/repository/kubespray/kubespray) and [quay.io/kubespray/vagrant:vX.Y.Z](https://quay.io/repository/kubespray/vagrant) docker images are built and tagged
-8. The `KUBESPRAY_VERSION` variable is updated in `.gitlab-ci.yml`
```

Comment:
https://github.com/kubernetes-sigs/kubespray/blob/master/.gitlab-ci.yml#L11 very much not removed. Just updated to latest release as it should be?

Reply:
Oh my mistake the Dockerfile change.. My bad will revert this